### PR TITLE
af_drc: remove

### PR DESCRIPTION
**Pull request by medhefgo**
Remove low quality drc filter. Anyone whishing to have dynamic range
compression should use the much more powerful acompressor ffmpeg filter:

	mpv --af=lavfi=[acompressor] INPUT

Or with parameters:

	mpv --af=lavfi=[acompressor=threshold=-25dB:ratio=3:makeup=8dB] INPUT

Refer to https://ffmpeg.org/ffmpeg-filters.html#acompressor for a full
list of supported parameters.

Or use https://github.com/medhefgo/mpv-acompressor for a lua script
that allows for changing parameters at runtime.

I agree that my changes can be relicensed to LGPL 2.1 or later.

### CODE REVIEW

File: TOOLS/lua/drc-control.lua [hunk 1]
```diff
@@ -1,99 +0,0 @@
...
-
-function drc_quieter_handler()
-    drc_scale_target(0.5)
-end
-
--- toggle between off, method 1 and method 2
-mp.add_key_binding("\\", "drc_toggle_method", drc_toggle_method_handler)
--- increase or decrease target volume
-mp.add_key_binding("ctrl+9", "drc_quieter", drc_quieter_handler)
-mp.add_key_binding("ctrl+0", "drc_louder", drc_louder_handler)
```
Comment (wm4) [inline TOOLS/lua/drc-control.lua]: I didn't even remember that we had this script. In this case, we could probably add yours (my understanding was you had one - I'm assuming it's similar to this one).

**Comment by haasn** [comment]
I'd also like to warmly recommend `loudnorm` and `dynaudnorm`.

**Comment by medhefgo** [comment]
I'd be happy to add my lua script to the patch if it's wanted. You can have a peek at https://github.com/medhefgo/mpv-acompressor (and ignore the licence :D).

And yeah, there's dynaudnorm. Personally, it's not that good with movies. The silent parts are nicely boosted, but the dynamics in the loud parts feel very off in comparison. Imho, it's better suited for music. Also, it doesn't cope at all with seeking back, completely forgetting about the loudness it previously assigned.
And thanks for letting me know about loudnorm.

File: TOOLS/lua/drc-control.lua (hunk 1 above)
Comment (kevmitch) [inline TOOLS/lua/drc-control.lua]: I added this script when lua examples were scarce and before I realized how horrible this filter was. Go ahead and nuke it from orbit.

**Comment by medhefgo** [comment]
I force pushed a new version that adds the script to the tools lua script collection.

**Comment by wm4** [comment]
Any opinions about the added script?

**Comment by wm4** [comment]
All merged, also added a mention to interface-changes.rst.

**Comment by nickcheng** [comment]
I installed MPV through HomeBrew and found there is no sound while playing video. By searching from the Internet, I came here and found the solution.

Since the filter was removed, I think configuration file should also been revised.

**Comment by wm4** [comment]
Which configuration file?

**Comment by nickcheng** [comment]
The default `.conf` file in `~/.config/mpv/mpv.conf`.

**Comment by wm4** [comment]
That's not the default config file, that's the one you put there.

**Comment by nickcheng** [comment]
Oh god! I thought it's default is because this is a newly installed laptop.

...but... you remind me. Probably the config file came back from my dot files backup. 🤦‍♂️

**Comment by medhefgo** [comment]
This must be something else. If I hadd "drc" to my mpv.conf, it will simply get ignored and audio plays properly.

**Pull request closed without merging**